wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationSenderMessageDetailsCell.swift

```diff
@@ -176,9 +174,7 @@ final class ConversationSenderMessageDetailsCell: UIView, ConversationMessageCel
             avatar.heightAnchor.constraint(equalToConstant: CGFloat(avatar.size.rawValue)),
 
             avatarTopAnchorConstraint,
-            avatarBottomAnchorConstraint,
             avatar.topAnchor.constraint(greaterThanOrEqualTo: topAnchor),
-            bottomAnchor.constraint(greaterThanOrEqualTo: avatar.bottomAnchor),
```

Comment:
Allows the avatar image to escape the bounds of its superview.
<img width="757" alt="Screenshot 2025-03-20 at 15 11 43" src="https://github.com/user-attachments/assets/4d5481a4-1484-4d6b-bf6f-eeaee0e0a76d" />